src/Context/Network/NetworkContext.jsx

```diff
@@ -20,6 +28,46 @@ const NetworkProvider = ({ children }) => {
 
   const googleProvider = new GoogleAuthProvider();
   const facebookProvider = new FacebookAuthProvider();
+
+  // add user to database after log in or signup
+  // const addUserToFirestore = async (userData) => {
+  //   const addUser = doc(firestore, `users/${userData.email}`);
+  //   try {
+  //     await setDoc(addUser, userData, { merge: true });
+  //   } catch (error) {
+  //     alertDispatchHandler(alertDispatch, "ALERT", "INFO", error.message);
+  //   }
+  // };
+
```

Comment:
Remove the commented code before pushing code to production

Reply:
Noted